### ISSUE TITLE
Stand alone version?

### ISSUE THREAD

**Issue by cypher27** [nicupavel/openpanzer#63]
First I will say that I am not a programmer by any means, aside from a little bit of HTML I learned years ago in High School.
But I was wondering why, if the whole source is right here in a .zip, why it wont work on its own when extracted in whole to its own file and opened domestically via chrome or firefox. Especially since it only depends on JavaScript and CSS. Am I missing something here? Is it purposed that we cannot run it on our own computer domestically?
I would like to play without needing an internet connection with my laptop on the go, and also tinker with some settings like starting prestige in a scenario just for fun.

**Comment by cypher27** [comment]
extracted to its own FOLDER*

**Comment by nicupavel** [comment]
It does work offline. Running the code locally in Google Chrome requires --allow-file-access-from-files option when starting Google Chrome (from README).

**Comment by cypher27** [comment]
Ah I see. Thank you for the quick reply! I will try that next time I play.

**Comment by cypher27** [comment]
I can't seem to figure out what you to start it that way with chrome. Ive tried my own hand at the command prompt and googled it but I still cant figure it out.

**Comment by nicupavel** [comment]
Theoretically you just open index.html in browser, I think firefox doesn't have the local file access restriction.

**Comment by cypher27** [comment]
Yes, as is why I can't figure it out. I've also tried tried firefox long before I posted on here.

Is it possible that some files are missing? On my last attempt to open the game in Chrome I opened up the part that views code and it had a number of File Not Found errors. Most notably for .wav sound files and the tutorial's scenario .xml

I checked the code you have posted here and where I believe should be the scenario xmls (following the path described in the error) and there is only a .gitignore file.

When I do open the index.html up locally via chrome or Firefox all I see are pretty much the the Title "OPEN PANZER" in its correct font as well as the background for the menu and the black background for the information up top. Which tells me atleast the html and CSS are working.
No map is loaded nor are any units or other information. Also no menu options just the background. 

Also your awesome romanian flag and maus tank combo info bar that's normally there is also missing :(

**Comment by cypher27** [comment]
It has now dawned on me that perhaps these files aren't actually missing but unavailable to download perhaps by intent. I realized this once I googled what a .gitignore file is.
If it is your purpose not to disclose files such as scenario xmls than forgive me for stepping on your toes there. Without them I am unable to play locally. I was hoping to maybe even fix some scenario bugs myself like with operation typhoon.

Anyhow great game you have made and improved here. I do appreciate your quick replies and all of your hard work.

**Comment by nicupavel** [comment]
Yes they were removed from the git because of some unfair usage. For 2.9 looks like I didn't generate an offline package, I will find some time to make one.

**Comment by cypher27** [comment]
Thank you. I look forward to the offline package and next update.

**Comment by alexvong1995** [comment]
Hi @nicupavel ,
What do you mean by unfair usage?
Do you mean the game data is generated from pg2, thus not distributable?
What do I need to do to make an offline version?
Thanks for your help.

**Comment by sprinkler** [comment]
By unfair usage I mean generating mobile apps from the available code and data, and selling those on Apple/Android stores.
I have a standalone version compatible with Chrome ready, I'm just gathering bug reports to fix remaining scenario issues. This version will work offline.

**Comment by nicupavel** [comment]
By unfair usage I mean generating mobile apps from the available code and data, and selling those on Apple/Android stores.
I have a standalone version compatible with Chrome ready, I'm just gathering bug reports to fix remaining scenario issues. This version will work offline.

**Comment by alexvong1995** [comment]
@nicupavel I see. While the GPL do allow commercial usage, it is a GPL violation if the guy selling it does not provide the modified source code or it is impossible to compile the modified source code. Besides, I think it is morally wrong by not mentioning you as the original author.

I suggest you to consult the [Software Freedom Conservancy](https://sfconservancy.org/), which is a not-for-profit charity that does GPL enforcement for FLOSS projects. I could help too if you are busy, but of course it is up to you to decide whether you want GPL enforcement since you are the copyright holder.

Finally, thanks for the game. It is valuable. I could only find 3 games of the same sort, lgeneral, open general and open panzer. Lgeneral works but is quite old, open general only works on windows, so open panzer is the way to go!
